daemon/info_unix.go

```diff
@@ -56,20 +56,23 @@ func (daemon *Daemon) FillPlatformInfo(v *types.Info, sysInfo *sysinfo.SysInfo)
 
 	v.InitCommit.Expected = dockerversion.InitCommitID
 	if rv, err := exec.Command(DefaultInitBinary, "--version").Output(); err == nil {
+		// examples of how Tini outputs version info:
+		//   "tini version 0.13.0 - git.949e6fa"
+		//   "tini version 0.13.2"
 		parts := strings.Split(strings.TrimSpace(string(rv)), " - ")
```

Comment:
That'd be an excellent idea IMO!  I can't work on that right away (so if someone else has bandwidth and cares, feel free to take over!), but I'll refactor as soon as I can if someone else doesn't. :smile: :heart: